### PR TITLE
Fenix youtube qoe

### PR DESCRIPTION
**Pull request by marianrai**
Besides the new craft method, I also added 2 changes for .taskcluster.yml and tasks.py to be able to test my commit before landing. I assume i have to remove those after testing.

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -145,7 +145,7 @@ tasks:
                           git fetch ${repository} ${head_branch}
                           && git config advice.detachedHead false
                           && git checkout FETCH_HEAD
-                          && python automation/taskcluster/decision_task.py pull-request
+                          && python automation/taskcluster/decision_task.py raptor
```
Comment (JohanLorenzo) [inline .taskcluster.yml]: To be 100% explicit: this change should be reverted 🙂

File: automation/taskcluster/lib/tasks.py [hunk 2]
```diff
@@ -441,15 +441,7 @@ def craft_raptor_signing_task(
...
-        routes = [
-            "index.project.mobile.fenix.v2{}.performance-test.{}.{}.{}.latest.{}".format(
-                staging_prefix, self.date.year, self.date.month, self.date.day, variant.abi
-            ),
-            "index.project.mobile.fenix.v2{}.performance-test.{}.{}.{}.revision.{}.{}".format(
-                staging_prefix, self.date.year, self.date.month, self.date.day, self.commit, variant.abi
-            ),
-            "index.project.mobile.fenix.v2{}.performance-test.latest.{}".format(staging_prefix, variant.abi),
-        ]
+        routes = []
```
Comment (JohanLorenzo) [inline automation/taskcluster/lib/tasks.py]: Same here

File: automation/taskcluster/lib/tasks.py [hunk 3]
```diff
@@ -561,6 +553,20 @@ def craft_function(signing_task_id, mozharness_task_id, variant, gecko_revision,
             )
         return craft_function
 
+    def craft_raptor_youtube_playback_task(self, signing_task_id, mozharness_task_id, variant, gecko_revision, force_run_on_64_bit_device=False):
```
Comment (davehunt) [inline automation/taskcluster/lib/tasks.py]: We need to exclude tests for resolutions above 1080p on Moto G5, like we did in taskcluster via [bug 1558456](https://bugzilla.mozilla.org/show_bug.cgi?id=1558456). It looks like we could use `variant` to determine the target device. Perhaps we could add an optional argument to `_craft_raptor_task` that takes a list of additional command line options.

Comment (marianrai) [reply automation/taskcluster/lib/tasks.py]: Hi Dave,
I added the logic from bug 1558456 in this commit:
https://github.com/mozilla-mobile/fenix/pull/4430/commits/55a76b8aa47ca771022ed3cdafd5c999e80b6e49

File: .taskcluster.yml (hunk 1 above)
Comment (marianrai) [reply .taskcluster.yml]: Hi Johan,
I reverted the changes in this commit: https://github.com/mozilla-mobile/fenix/pull/4430/commits/55a76b8aa47ca771022ed3cdafd5c999e80b6e49

I also did a rebase and now i see that i have 42 commits from other people. Let me know if i should cancel the pull request and start a new one.

File: automation/taskcluster/lib/tasks.py (hunk 2 above)
Comment (marianrai) [reply automation/taskcluster/lib/tasks.py]: I reverted the changes in this commit:
https://github.com/mozilla-mobile/fenix/pull/4430/commits/55a76b8aa47ca771022ed3cdafd5c999e80b6e49

File: .taskcluster.yml (hunk 1 above)
Comment (JohanLorenzo) [reply .taskcluster.yml]: Yeah, it's probably easier to start over.

**Comment by marianrai** [comment]
I closed this one because it's a total mess in my branch.
I'll start it over.

**Pull request closed without merging**